packages/fxa-auth-server/lib/senders/emails/bin/mjml-sever.ts

```diff
@@ -0,0 +1,56 @@
+import http from 'http';
+import stream from 'stream';
+import net from 'net';
+import { render } from '../renderer';
+
+const { PORT = 8192, HOST = '0.0.0.0' } = process.env;
+
+const MJML_OPTIONS = {};
+
+console.log(`Starting MJML API server at ${HOST}:${PORT}`);
+
+const server = http.createServer(handleRequest);
+server.on('clientError', handleError);
+server.listen({ port: PORT, host: HOST });
+
+async function handleRequest(
+  req: http.IncomingMessage,
+  res: http.ServerResponse
+) {
+  try {
+    const body = await readStream(req);
+    const data = JSON.parse(body.trim());
+    const { template: templateName, ...variables } = data;
+
+    console.log(`REQUEST ${templateName}`);
```

Comment:
I don't mind leaving this in if it helps for now but we may want to remove it when it becomes unhelpful (which is probably soon if not now, since this is hit for every story).